migrations/1718888344202-seedNotificationTypeForNotifyRewardAmount.ts

```diff
@@ -0,0 +1,27 @@
+import { MigrationInterface, QueryRunner } from "typeorm"
+import { NOTIFICATION_CATEGORY, NOTIFICATION_TYPE_NAMES } from '../src/types/general';
+import { MICRO_SERVICES } from '../src/utils/utils';
+import { NotificationType, SCHEMA_VALIDATORS_NAMES } from '../src/entities/notificationType';
+
+const NotifyRewardAmountNotificationType = [
+    {
+        name: NOTIFICATION_TYPE_NAMES.NOTIFY_REWARD_AMOUNT,
```

Comment:
@mohammadranjbarz 
I ran the migration and these fields are set successfully, but I am not sure about the correctness of their values and I don't now that I should set other fields too, or not